website/docs/r/backup_compliance_policy.html.markdown

```diff
@@ -86,6 +86,11 @@ resource "mongodbatlas_cloud_backup_schedule" "test" {
     retention_unit     = "months"
     retention_value    = 12
   }
+  policy_item_yearly {
```

Comment:
`### Policy Item Yearly` should be added in this file also

Reply:
thanks, added